src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -218,6 +219,37 @@ private function addCsrfSection(ArrayNodeDefinition $rootNode): void
         ;
     }
 
+    private function addClockSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('clock')
+                    ->info('Clock configuration')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('default_timezone')->defaultValue(null)
```

Comment:
please wrap `->defaultNull()` on its own line for consistency. And please use the dedicated method

Reply:
Indeed, thanks!